drivers/gles3/storage/render_scene_buffers_gles3.cpp

```diff
@@ -131,6 +131,7 @@ void RenderSceneBuffersGLES3::configure(const RenderSceneBuffersConfiguration *p
 	scaling_3d_mode = p_config->get_scaling_3d_mode();
 	//fsr_sharpness = p_config->get_fsr_sharpness();
 	//texture_mipmap_bias = p_config->get_texture_mipmap_bias();
+	//anisotropic_filtering_level = p_config->get_anisotropic_filtering_level();
```

Comment:
Is there a purpose to the commented code being introduced here?

Reply:
While FSR1 probably won't be implemented in Compatibility, anisotropic filtering level change can be done on all platforms (including OpenGL ES) while mipmap LOD bias is limited to desktop OpenGL (it was never part of OpenGL ES).